src/Data/Skills/act_dex.lua

```diff
@@ -7400,6 +7417,147 @@ skills["PhysCascadeTrap"] = {
+			local fullRadius = output.AreaOfEffectRadius
+			dpsMultiplier = maxWaves * hitChance(enemyRadius, waveRadius, fullRadius)
+			if breakdown then
+				local breakdownDpsMult = {}
+				t_insert(breakdownDpsMult, "DPS multiplier")
+				t_insert(breakdownDpsMult, s_format("^8= ^7maximum waves^8 * (^7secondary radius^8 + ^7enemy radius^8 - 1) ^ 2 / ^7radius^8 ^ 2"))
+				t_insert(breakdownDpsMult, s_format("^8= ^7%d^8 * (^7%d^8 + ^7%d^8 - 1)^2 / ^7%d^8^2", maxWaves, waveRadius, enemyRadius, fullRadius))
+				t_insert(breakdownDpsMult, s_format("^8= ^7%.3f", dpsMultiplier))
+				breakdown.SkillDPSMultiplier = breakdownDpsMult
+			end
+		elseif skillPart == 3 then
+			dpsMultiplier = maxWaves
+			if breakdown then
+				local breakdownDpsMult = {}
+				t_insert(breakdownDpsMult, "DPS multiplier")
+				t_insert(breakdownDpsMult, s_format("^8= ^7%d (maximum waves)", dpsMultiplier))
+				breakdown.SkillDPSMultiplier = breakdownDpsMult
+			end
+		elseif skillPart == 4 then
+			dpsMultiplier = averageActiveTraps
+			if breakdown then
+				local breakdownDpsMult = {}
+				t_insert(breakdownDpsMult, "DPS multiplier")
+				t_insert(breakdownDpsMult, s_format("^8= ^7%.2f (average active traps)", dpsMultiplier))
+				breakdown.SkillDPSMultiplier = breakdownDpsMult
+			end
+		elseif skillPart == 5 then
+			local enemyRadius = skillModList:Sum("BASE", skillCfg, "EnemyRadius")
+			local waveRadius = output.AreaOfEffectRadiusSecondary
+			local fullRadius = output.AreaOfEffectRadius
+			dpsMultiplier = averageActiveTraps * maxWaves * hitChance(enemyRadius, waveRadius, fullRadius)
+			if breakdown then
+				local breakdownDpsMult = {}
+				t_insert(breakdownDpsMult, "DPS multiplier")
+				t_insert(breakdownDpsMult, "^8= ^7average active traps^8 * ^7maximum waves^8 * (^7secondary radius^8 + ^7enemy radius^8 - 1) ^ 2 / ^7radius^8 ^ 2")
+				t_insert(breakdownDpsMult, s_format("^8= ^7%.2f^8 * ^7%d^8 * (^7%d^8 + ^7%d^8 - 1) ^ 2 / ^7%d^8 ^ 2", averageActiveTraps, maxWaves, waveRadius, enemyRadius, fullRadius))
+				t_insert(breakdownDpsMult, s_format("^8= ^7%.3f", dpsMultiplier))
+				breakdown.SkillDPSMultiplier = breakdownDpsMult
+			end
+		elseif skillPart == 6 then
+			dpsMultiplier = averageActiveTraps * maxWaves
+			if breakdown then
+				local breakdownDpsMult = {}
+				t_insert(breakdownDpsMult, "DPS multiplier")
+				t_insert(breakdownDpsMult, "^8= ^7average active traps^8 * ^7maximum waves")
+				t_insert(breakdownDpsMult, s_format("^8= ^7%.2f^8 * ^7%d", averageActiveTraps, dpsMultiplier))
+				breakdown.SkillDPSMultiplier = breakdownDpsMult
+			end
+		end
+		if dpsMultiplier ~= 1 then
+			skillData.dpsMultiplier = (skillData.dpsMultiplier or 1) * dpsMultiplier
+			output.SkillDPSMultiplier = (output.SkillDPSMultiplier or 1) * dpsMultiplier
+		end
+	end,
+	statMap = {
+		["base_skill_show_average_damage_instead_of_dps"] = {},
+		["phys_cascade_trap_base_interval_duration_ms"] = {
+			skill("repeatFrequency", nil),
```

Comment:
```suggestion
			skill("PulseCooldown", nil),
```
Most mods are capitalised like this and this is the time between pulses not the frequency
Pulse rate should probably be changed to be the inverse of how you currently have it or relaballed to pulse interval or similar.

Reply:
Yeah fair enough on the capitalisation pick which ever then.